plugins/woocommerce/tests/php/src/Gateways/PayPal/RequestTest.php

```
@@ -522,6 +522,49 @@ function ( $value, $parsed_args, $url ) use ( &$capture_api_call_count ) {
+	 *
+	 * @return void
+	 */
+	public function test_capture_authorized_payment_not_attempted_when_order_details_404(): void {
+		$order = \WC_Helper_Order::create_order();
+		$order->update_meta_data( PayPalConstants::PAYPAL_ORDER_META_ORDER_ID, 'PAYPAL_ORDER_123' );
+		$order->save();
+
+		$order_details_call_count = 0;
+		add_filter(
+			'pre_http_request',
+			function ( $value, $parsed_args, $url ) use ( &$order_details_call_count ) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$parsed_args'. (undefined)

(UnusedFormalParameter)